cypress-tests/cypress/commands/commands.js

```diff
@@ -425,19 +421,7 @@ Cypress.Commands.add("getPosition", (componentName) => {
 });
 
 Cypress.Commands.add("defaultWorkspaceLogin", () => {
```

Comment:
Please remove this and replace this login with apilogin and additional assertion if nessoery.
For me it looks like unnessory command, right now.